app/serializers/webhook/article_serializer.rb

```diff
@@ -0,0 +1,38 @@
+module Webhook
+  class ArticleSerializer
+    extend ApplicationHelper
```

Comment:
I'm not super-satisfied with extending the serializer with these helpers, maybe we could refactor this.

Reply:
Can't you just explicitly call the functions you need instead of including the whole thing?

Reply:
What do you mean by calling? They are called inside the serializer, but we need to include them somehow before calling.